lms/djangoapps/django_comment_client/forum/views.py

```diff
@@ -54,7 +54,7 @@ class DiscussionTab(EnrolledTab):
     """
 
     type = 'discussion'
-    title = ugettext_noop('Discussion')
+    title = pgettext('Courseware discussion tab', 'Discussion')
```

Comment:
I think this won't be translated, so let's put a context here because the original is a `ugettext_noop` and the latter mean that the actual translation is being done somewhere else!

Please put the context on [the other ones](https://github.com/Edraak/edx-platform/blob/master/cms/djangoapps/contentstore/views/component.py#L187).